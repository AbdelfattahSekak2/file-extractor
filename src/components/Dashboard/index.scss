@import "../../styles/mixins";
@import url("https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap");

.dashboard-container {
  display: flex;
  flex-direction: row;
  background: $light-grey;
  padding: 20px;
  height: 100vh;
  font-family: "Roboto", sans-serif;
}
