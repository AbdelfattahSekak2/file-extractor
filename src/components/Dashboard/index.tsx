import * as React from "react";

import "./index.scss";

import UploderContainer from "../../containers/UploderContainer";
import ExtractionContainer from "../../containers/ExtractionContainer";
import ErrorContainer from "../../containers/ErrorContainer";

const Dashboard: React.FunctionComponent = () => (
  <div className="dashboard-container">
    <div className="dashboard-title"><strong>Mindee</strong> File Extractor</div>
    <div className="content">
      <ErrorContainer />
      <UploderContainer />
      <ExtractionContainer />
    </div>
  </div>
)

export default Dashboard;

