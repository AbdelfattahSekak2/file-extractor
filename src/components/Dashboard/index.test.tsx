import React from "react";
import { ShallowWrapper, shallow } from "enzyme";

import Dashboard from ".";

describe("Dashboard component", () => {
  let wrapper: ShallowWrapper = shallow(<Dashboard />)
  it("should render correctly", () => {
    expect(wrapper).not.toBeNull();
    expect(wrapper).toMatchSnapshot();
  })
})
