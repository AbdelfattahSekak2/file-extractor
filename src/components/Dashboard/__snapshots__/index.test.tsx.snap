// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard component should render correctly 1`] = `
<div
  className="dashboard-container"
>
  <Connect(ErrorMessage) />
  <Connect(Uploader) />
  <Connect(ExtractionView) />
</div>
`;
